frontend/src/bundles/common/components/form-control/form-control.tsx

```diff
@@ -0,0 +1,46 @@
+import { type SxProps, type Theme } from '@mui/material';
+import { FormControl as MUIFormControl } from '@mui/material';
+
+const InputVariantTypes = {
+    FILLED: 'filled',
+    OUTLINED: 'outlined',
+    STANDART: 'standard',
+} as const;
+
+type FormControlVariants =
+    (typeof InputVariantTypes)[keyof typeof InputVariantTypes];
+
+type Properties = {
+    children: React.ReactNode;
+    color?: 'primary' | 'secondary' | 'error' | 'info' | 'success' | 'warning'; // TODO: change after the color scheme is merged
+    variant?: FormControlVariants;
+    isError?: boolean;
+    isDisabled?: boolean;
+
+    className?: string;
+    sx?: SxProps<Theme>;
+};
+
+const FormControl: React.FC<Properties> = ({
+    children,
+    color = 'primary',
+
+    variant = 'standard',
```

Comment:
In this case you should use your object values:

```suggestion

    variant = InputVariantTypes.STANDART,
```